src/components/side-suggested-users.js

```diff
@@ -47,6 +50,32 @@ export default class SideSuggestedUsers extends React.Component {
     }))
   };
 
+  ignoreUser = async (user) => {
+    const elems = document.querySelectorAll('.suggested_users');
+    _.each(elems, c => c.style.opacity = '');
+
+    animate(
+      firstTimePassed => {
+        _.each(elems,
+          c => c.style.opacity = 1 / (1 + firstTimePassed*0.1)
+        );
+
+      }, 70, async () => {
+        await this.props.triggers.ignoreUser(user);
+
+        animate(
+          secondTimePassed => {
+            _.each(elems,
+              c => c.style.opacity = 1 / (1 + (70 - secondTimePassed)*0.1)
+            );
+          }, 70, () => {
```

Comment:
again: extract callback into variable.

we want to keep `animate()` call short and concise on a single line